src/raft.rs

```diff
@@ -317,7 +317,10 @@ pub fn vote_resp_msg_type(t: MessageType) -> MessageType {
 
 // Calculate the quorum of a Raft cluster with the specified total nodes.
 pub fn quorum(total: usize) -> usize {
-    total / 2 + 1
+    match total {
```

Comment:
I think the original implement is almost OK. For 0, it should panic instead.

Reply:
Indeed you are right, only the 0-case was off. Perhaps I should just close this PR and wait to use [std::num::NonZeroUsize](https://doc.rust-lang.org/1.26.0/std/num/struct.NonZeroUsize.html) once it reaches stable (in 1.28.0)?

Reply:
Interesting. `NonZeroUsize` sounds like a good idea to me.

Reply:
Ack. I'm closing this, I'll keep a note to revisit after 1.28.0 release.